### PR TITLE
[stable/postgresql] Add extraVolumeMount and Volume to Postgresql Master and Slave

### PR DESCRIPTION
**Pull request by baluchicken**
Signed-off-by: Balint Molnar <balintmolnar91@gmail.com>

#### What this PR does / why we need it:
This PR allows to add extraVolumeMounts and Volumes to Postgresql Master and Slave StatefulSet.
E.g.:
```
  extraVolumeMounts:
    - name: bar
      mountPath: /bar/
    - name: foo
      mountPath: /foosecret/
  extraVolumes:
    - name: foo
      secret:
        secretName: foosecret
    - name: bar
      emptyDir: {}
```
#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/postgresql/templates/statefulset-slaves.yaml [hunk 1]
```diff
@@ -203,6 +206,9 @@ spec:
       - name: data
         emptyDir: {}
       {{- end }}
+      {{- if .Values.slave.extraVolumes }}
+      {{ toYaml .Values.slave.extraVolumes | indent 6}}
```
Comment (javsalgar) [inline stable/postgresql/templates/statefulset-slaves.yaml]:
```suggestion
      {{- toYaml .Values.slave.extraVolumes | nindent 6}}
```

File: stable/postgresql/templates/statefulset-slaves.yaml [hunk 2]
```diff
@@ -183,6 +183,9 @@ spec:
         - name: postgresql-config
           mountPath: /bitnami/postgresql/conf
         {{- end }}
+        {{- if .Values.slave.extraVolumeMounts }}
+        {{ toYaml .Values.slave.extraVolumeMounts | nindent 8 }}
```
Comment (javsalgar) [inline stable/postgresql/templates/statefulset-slaves.yaml]:
```suggestion
        {{- toYaml .Values.slave.extraVolumeMounts | nindent 8 }}
```

File: stable/postgresql/templates/statefulset-slaves.yaml (hunk 1 above)
Comment (javsalgar) [reply stable/postgresql/templates/statefulset-slaves.yaml]: I see that in one it is indent 6 and in the other indent 8. Make sure which the right indentation is

**Review by javsalgar — CHANGES_REQUESTED** [review]

**Comment by baluchicken** [comment]
@javsalgar thanks for your review in case of extraVolume it is 6 but for extraVolumeMount it is 8

**Comment by javsalgar** [comment]
LGTM. Could you please fix the conflicts?

**Comment by baluchicken** [comment]
@javsalgar rebased to master

**Pull request merged**